modules/atomic-opt-in/module.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Elementor\Modules\AtomicOptIn;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+
+use Elementor\Plugin;
+use Elementor\Settings;
+use Elementor\Utils;
+
+class Module extends BaseModule {
+	const MODULE_NAME = 'editor-v4-opt-in';
+	const EXPERIMENT_NAME = 'editor_v4';
+
+	public static function get_experimental_data(): array {
+		return [
+			'name' => self::EXPERIMENT_NAME,
+			'title' => esc_html__( 'Editor V4', 'elementor' ),
+			'description' => esc_html__( 'Enable Editor V4.', 'elementor' ),
+			'hidden' => true,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+			'release_status' => Experiments_Manager::RELEASE_STATUS_ALPHA,
+		];
+	}
+
+	private function is_experiment_active() {
+		return Plugin::$instance->experiments->is_feature_active( self::EXPERIMENT_NAME );
+	}
+
+	public function __construct() {
+		if ( ! $this->is_experiment_active() || ! is_admin() ) {
+			return;
+		}
+
+		$this->register_settings_tab();
+		$this->register_scripts();
+	}
+
+	public function get_name() {
+		return self::MODULE_NAME;
+	}
+
+	private function register_settings_tab() {
+		$page_id = Settings::PAGE_ID;
+
+		add_action( "elementor/admin/after_create_settings/{$page_id}", function( Settings $settings ) {
+			$this->add_new_tab_to( $settings );
+		}, 11 );
+	}
+
+	private function add_new_tab_to( Settings $settings ) {
+		$settings->add_tab( self::MODULE_NAME, [
+			'label' => esc_html__( 'Editor V4', 'elementor' ),
+			'sections' => [
+				'opt-in' => [
+					'callback' => function() {
+						echo '<div id="page-editor-v4-opt-in"></div>';
```

Comment:
Maybe?

```suggestion
						echo '<div id="e-settings-tab-v4-opt-in"></div>';
```

Reply:
In my experience in 99% of the cases these promotion type of files are hardly ever updated, but we might need to add it at some stage.